alf/networks/projection_networks.py

```diff
@@ -34,6 +34,8 @@ def __init__(self,
                  input_size,
                  action_spec,
                  logits_init_output_factor=0.1,
+                 weight_opt_args=None,
+                 bias_opt_args=None,
```

Comment:
Why are these arguments only added for this class but not for the other ones?

Reply:
No particular reason. It's just that I didn't use the others in experiments.